### PR TITLE
🚨🚨 Source Pipedrive: Remove `followers_count` field from `Products` stream 

### PR DESCRIPTION
**Pull request by artem1205**
## What

Fix Source Pipedrive Connector Health

## How

- Fix expected records;
- update schema Products (Reason: [PipeDrive Changelog](https://devcommunity.pipedrive.com/t/removal-of-followers-count-response-field-from-products-endpoints-and-webhook/6135))

## Recommended reading order
1. `expected_records.jsonl`
2. `products.json`

## 🚨 User Impact 🚨

`Products` schema changed; schema update required

Playbook: https://docs.google.com/document/d/16rwD7XEstml1TiW8PH6mjk00q0vynlHDL0TvWPEG0io/edit

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-pipedrive test report (commit [`9be7546d40`](https://github.com/airbytehq/airbyte/pull/27832/commits/9be7546d40e618be3e288d7bd0f398deea75e45c)) - ❌

⏲️  Total pipeline duration: 05mn03s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-pipedrive/metadata.yaml | ✅        |
| Connector version semver check                                          | ✅        |
| Connector version increment check                                       | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-pipedrive docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Integration tests                                                       | ✅        |
| Acceptance tests                                                        | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-pipedrive/1688047667/9be7546d40e618be3e288d7bd0f398deea75e45c/source-pipedrive/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pipedrive test
```

**Comment by octavia-squidington-iii** [comment]
##  source-pipedrive test report (commit [`9be7546d40`](https://github.com/airbytehq/airbyte/pull/27832/commits/9be7546d40e618be3e288d7bd0f398deea75e45c)) - ✅

⏲️  Total pipeline duration: 04mn28s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-pipedrive/metadata.yaml | ✅        |
| Connector version semver check                                          | ✅        |
| Connector version increment check                                       | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-pipedrive docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Integration tests                                                       | ✅        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-pipedrive/1688048078/9be7546d40e618be3e288d7bd0f398deea75e45c/source-pipedrive/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pipedrive test
```

**Review by alafanechere — COMMENTED** [review]
@artem1205 does keeping the field in the schema is causing a sync issue?

**Comment by artem1205** [comment]
> @artem1205 does keeping the field in the schema is causing a sync issue?

No, as we don't receive it in response.

**Comment by artem1205** [comment]
> Would you mind publishing a pre-release and try it on a connection with the previous version to check if the column is deleted on the destination side? I believe normalization to be non destructive so hopefully the destination column is not deleted. Moreover if schema change is detected the user will be notified about this change. If it's still available on the destination we can move forward without a breaking change process

May I check this locally using Postgres?

File: docs/integrations/sources/pipedrive.md [hunk 1]
```diff
@@ -111,7 +111,8 @@ The Pipedrive connector will gracefully handle rate limits. For more information
 ## Changelog
 
 | Version | Date       | Pull Request                                             | Subject                                                        |
-| :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------- |
+|:--------|:-----------|:---------------------------------------------------------|:---------------------------------------------------------------|
+| 1.0.0   | 2023-06-29 | [27832](https://github.com/airbytehq/airbyte/pull/27832) | Update `Products` schema                                       |
```
Comment (evantahler) [inline docs/integrations/sources/pipedrive.md:115]: Please explain why we are updating the schema.  What field was dropped and why?

Comment (evantahler) [reply docs/integrations/sources/pipedrive.md:115]: Similarly, this PRs title and description can explain the change better

Comment (artem1205) [reply docs/integrations/sources/pipedrive.md:115]: During my work on connector health report, i've mentioned that there is no `followers_count` field in response.
After some googling i've found [an announcement ](https://devcommunity.pipedrive.com/t/removal-of-followers-count-response-field-from-products-endpoints-and-webhook/6135) in PipeDrive Changelog.

Would you like me to rename changes to """ Remove  `followers_count` field from `Products` stream""" ?

Comment (evantahler) [reply docs/integrations/sources/pipedrive.md:115]: yes please!

**Comment by artem1205** [comment]
@alafanechere

### Actions Made

1. Sync with version `0.1.18`
2. bumping to `dev`
3. schema refresh (both with resetting and without)
4. sync using `dev` version

### Test Results

column is deleted

**Comment by octavia-squidington-iii** [comment]
##  source-pipedrive test report (commit [`46319fbb74`](https://github.com/airbytehq/airbyte/pull/27832/commits/46319fbb74fc4e45f46f130ccf8514370a798cf1)) - ❌

⏲️  Total pipeline duration: 05mn11s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-pipedrive/metadata.yaml | ✅        |
| Connector version semver check                                          | ✅        |
| Connector version increment check                                       | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-pipedrive docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Integration tests                                                       | ✅        |
| Acceptance tests                                                        | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-pipedrive/1688065866/46319fbb74fc4e45f46f130ccf8514370a798cf1/source-pipedrive/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pipedrive test
```

**Comment by octavia-squidington-iii** [comment]
##  source-pipedrive test report (commit [`1d1e7e41c0`](https://github.com/airbytehq/airbyte/pull/27832/commits/1d1e7e41c08175e4e8d41225d540aaccf652f863)) - ❌

⏲️  Total pipeline duration: 04mn07s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-pipedrive/metadata.yaml | ✅        |
| Connector version semver check                                          | ✅        |
| Connector version increment check                                       | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-pipedrive docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Integration tests                                                       | ✅        |
| Acceptance tests                                                        | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-pipedrive/1688070519/1d1e7e41c08175e4e8d41225d540aaccf652f863/source-pipedrive/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pipedrive test
```

**Comment by octavia-squidington-iii** [comment]
##  source-pipedrive test report (commit [`18b8c12969`](https://github.com/airbytehq/airbyte/pull/27832/commits/18b8c12969bde37c94ca34f89b03bf2c78cab8cc)) - ✅

⏲️  Total pipeline duration: 04mn18s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-pipedrive/metadata.yaml | ✅        |
| Connector version semver check                                          | ✅        |
| Connector version increment check                                       | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-pipedrive docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Integration tests                                                       | ✅        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-pipedrive/1688072267/18b8c12969bde37c94ca34f89b03bf2c78cab8cc/source-pipedrive/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pipedrive test
```

**Comment by alafanechere** [comment]
@artem1205 if the destination column was deleted by normalization please fill out start a breaking change procedure by creating a new document following this template: https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit#heading=h.r2xzw0kclodc

This is a breaking change because on connector upgrade the customer will lose data...

**Comment by octavia-squidington-iii** [comment]
##  source-pipedrive test report (commit [`69a504bbf0`](https://github.com/airbytehq/airbyte/pull/27832/commits/69a504bbf0972f382ab14a882c6f7c575c1def52)) - ❌

⏲️  Total pipeline duration: 05mn10s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-pipedrive/metadata.yaml | ✅        |
| Connector version semver check                                          | ✅        |
| Connector version increment check                                       | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-pipedrive docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Integration tests                                                       | ✅        |
| Acceptance tests                                                        | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-pipedrive/1688651859/69a504bbf0972f382ab14a882c6f7c575c1def52/source-pipedrive/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pipedrive test
```

**Review by lazebnyi — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-pipedrive test report (commit [`4e055ad8a0`](https://github.com/airbytehq/airbyte/pull/27832/commits/4e055ad8a0152688a656af459b9090e78a968241)) - ✅

⏲️  Total pipeline duration: 05mn09s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-pipedrive/metadata.yaml | ✅        |
| Connector version semver check                                          | ✅        |
| Connector version increment check                                       | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-pipedrive docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Integration tests                                                       | ✅        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-pipedrive/1688654934/4e055ad8a0152688a656af459b9090e78a968241/source-pipedrive/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-pipedrive test
```

**Pull request merged**